cyb_oko/settings.py

```diff
@@ -36,6 +36,7 @@
     'voucher',
     'z',
     'members',
+    'intern',
```

Comment:
Any reason we call the app `intern` and not e.g. `access`? Considering the branch is called `access` :p